src/index.js

```diff
@@ -228,14 +243,15 @@ class Tracing {
       recorder: options.recorder
     });
     this._Span = SpanCreator({
+      otTracer: this,
       tracer: this._zipkinTracer,
       serviceName: this._serviceName,
       kind: options.kind,
       sampler: options.sampler
     });
   }
 
-  startSpan(name, options = {}) {
+  _startSpan(name, options) {
```

Comment:
Why the underscores all over the place?

Reply:
That is how the `OpenTracing` base classes are defining the methods that need to be implemented in the derived classes, see:
* `tracer`: https://github.com/opentracing/opentracing-javascript/blob/d2c1c6a5a50439bd769d02c7f4638293db40140a/src/tracer.ts#L158-L180
* `span`: https://github.com/opentracing/opentracing-javascript/blob/d2c1c6a5a50439bd769d02c7f4638293db40140a/src/span.ts#L175-L221

Reply:
Okay, this seems to be fine 👍